x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_actions_column.tsx

```diff
@@ -42,67 +127,42 @@ export const getUseActionColumnHook =
       } = getAlertsDefaultModel(license),
     } = useSelector((state: State) => eventsViewerSelector(state, tableId));
 
-    const columnHeaders = columns;
-
     const renderCustomActionsRow = useCallback(
       ({
         rowIndex,
         cveProps,
         setIsActionLoading,
-        refresh: alertsTableRefresh,
+        refresh,
         clearSelection,
-        ecsAlert: alert,
+        ecsAlert,
         nonEcsData,
       }) => {
-        const timelineItem: TimelineItem = {
```

Comment:
New object reference. Not sure if it would cause re-renders as the reference, but this fn gets called multiple times for each row on a re-render. The issue may be in `triggers-actions-ui` though. See files further below